db/l2db/apiqueries.go

```diff
@@ -127,3 +129,153 @@ func (l2db *L2DB) GetTxAPI(txID common.TxID) (*PoolTxAPI, error) {
 		txID,
 	))
 }
+
+// GetPoolTxsAPIRequest is an API request struct for getting txs from the pool
+type GetPoolTxsAPIRequest struct {
+	EthAddr     *ethCommon.Address
+	FromEthAddr *ethCommon.Address
+	ToEthAddr   *ethCommon.Address
+	Bjj         *babyjub.PublicKeyComp
+	FromBjj     *babyjub.PublicKeyComp
+	ToBjj       *babyjub.PublicKeyComp
+	TxType      *common.TxType
+	Idx         *common.Idx
+	FromIdx     *common.Idx
+	ToIdx       *common.Idx
+	State       *common.PoolL2TxState
+
+	FromItem *uint
+	Limit    *uint
+	Order    string
+}
+
+// GetPoolTxs return Txs from the pool
+func (l2db *L2DB) GetPoolTxs(request GetPoolTxsAPIRequest) ([]PoolTxAPI, uint64, error) {
```

Comment:
Currently we have all the SQL DB functions that are used by the API ending with the prefix `API`.  This is useful to distinguish DB functions that acquire the semaphore (those that end with `API`) and those that don't, and then it's easier to avoid calling an API function from internal code, and a non-API function from the API.

So please, could you rename this function to `GetPoolTxsAPI`?